ffi/diplomat/src/data_struct.rs

```diff
@@ -2,19 +2,20 @@
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
-use alloc::borrow::Cow;
-use alloc::borrow::ToOwned;
+#[cfg(feature = "icu_decimal")]
+use alloc::borrow::{Cow, ToOwned};
 
 #[diplomat::bridge]
 pub mod ffi {
-    use super::str_to_cow;
+
+    #[cfg(feature = "icu_decimal")]
     use crate::errors::ffi::ICU4XError;
     use alloc::boxed::Box;
+    #[cfg(feature = "icu_decimal")]
     use icu_decimal::provider::{
```

Comment:
Suggestion (optional): move more of these imports into the function body so you can reduce the number of `cfg`s.